### DISCUSSION TITLE
Impossible to log when LDAP user has multiples e-mail

### DISCUSSION THREAD

**Discussion by Zykino** [requarks/wiki#3673] [Error / Bug Report]
**Description**
I’m installing wiki.js on my yunohost server using the package https://github.com/YunoHost-Apps/wikijs_ynh. When setting up the LDAP authentication I (re)discovered that the first user can not log on wiki.js with LDAP, but another user can. I tried adding other e-mails to the "second" user which was able to connect earlier and he couldn’t anymore.

**To Reproduce**
Steps to reproduce the behavior (Not sure how to configure LDAP usually):
1. Connect LDAP
2. Login as a user with only one e-mail configured (OK)
3. Disconnect
4. Add two more e-mails to the user
5. Try to connect
=> Get the same error as the first user: "Missing or invalid email address from profile."

**Expected behavior**
Connection succeed.

**Host Info (please complete the following information):**
 - OS: Debian 9.12 64bit (Linux 4.9.0-12-amd64)
 - Wiki.js version: 2.3.77
 - Database engine: ?

**Additional context**
Using Yunohost as said in the description.

**Comment by Mikrobit** [comment]
This patch fixes the issue for me: https://github.com/Requarks/wiki/pull/2515
